### PR TITLE
User auth reg login

### PR DESCRIPTION
**Pull request by nataliiaR**
landing page redesign, added index handlebars template, added facebook user auth and login redirect to Tell us your diet

### CODE REVIEW

File: models/user_info.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
```
Comment (KyleKnocke) [inline models/user_info.js]: Go ahead and merge this with userCredentials.js we done want these to be seperate models, 
Additionally userCredentials stores the encrypted password as well.

File: server.js [hunk 2]
```diff
@@ -1,22 +1,122 @@
-const express = require("express");
+
 const exphbs = require("express-handlebars");
+require('dotenv').config();
+const env = process.env.NODE_ENV || 'development';
+const express         =     require('express')
```
Comment (KyleKnocke) [inline server.js]: Go ahead and create a folder called access and add passport module js in that location. 
We dont its functionality in the body of server.js

Comment (nataliiaR) [reply server.js]: created a task for that

File: models/user_info.js (hunk 1 above)
Comment (nataliiaR) [reply models/user_info.js]: deleted that file all together, will use userCredentials.js

**Pull request merged**